apps/src/aiTutor/redux/aiTutorRedux.ts

```diff
@@ -1,26 +1,20 @@
-import {getChatCompletionMessage} from '@cdo/apps/aichat/chatApi';
+import {getChatCompletionMessage} from '@cdo/apps/aiTutor/chatApi';
 import {createSlice, PayloadAction, createAsyncThunk} from '@reduxjs/toolkit';
-import {
-  compilationSystemPrompt,
-  generalChatSystemPrompt,
-  validationSystemPrompt,
-} from '@cdo/apps/aiTutor/constants';
+import {systemPrompt} from '@cdo/apps/aiTutor/constants';
 import {savePromptAndResponse} from '../interactionsApi';
 import {
   Role,
   AITutorInteractionStatus as Status,
   ChatCompletionMessage,
   Level,
   ChatContext,
-  AITutorTypesValue,
-  AITutorTypes as TutorTypes,
   AITutorInteractionStatusValue,
+  AITutorTypes,
 } from '../types';
 
 const registerReducers = require('@cdo/apps/redux').registerReducers;
 
 export interface AITutorState {
-  selectedTutorType: AITutorTypesValue | undefined;
```

Comment:
Since there are no longer three different scenarios in the UI/there's only one tutor, we no longer need to track the selected tutor type. We only need to pass the tutor type/action chosen on the POST request to OpenAI, so we know when to include the validation code on the backend.